pkgs/games/xjump/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, gcc, autoconf, automake, xorg, ... }:
+
+stdenv.mkDerivation rec {
+  name = "xjump-${version}";
+  version = "2.9.3";
+  src = fetchFromGitHub {
+    owner = "hugomg";
+    repo = "xjump";
+    rev = "e7f20fb8c2c456bed70abb046c1a966462192b80";
+    sha256 = "0hq4739cvi5a47pxdc0wwkj2lmlqbf1xigq0v85qs5bq3ixmq2f7";
+  };
+  buildInputs = [ gcc autoconf automake xorg.libX11 xorg.libXt xorg.libXpm xorg.libXaw ];
```

Comment:
autoconf & automake should be nativeBuildInputs.

Consider replacing these (and the preConfigure attribute) with autoreconfHook (I haven't tested if this works).

Reply:
Oh--and why is "gcc" a buildInput? It almost never should be, especially in code that isn't itself a compiler or runtime of some sort.